lightning/src/onion_message/messenger.rs

```diff
@@ -396,7 +399,7 @@ where
 			},
 		};
 
-		log_trace!(self.logger, "Responding to onion message with path_id {:02x?}", path_id);
+		log_trace!(self.logger, "Responding to onion message {:?} with path_id {:02x?}", response, path_id);
```

Comment:
This makes it seem like the `response` is the message being responded to. Let's instead form this log as:

```rust
log_trace!(self.logger, "Responding to onion message with path_id {:02x?} with: {:?}", path_id, response);
```